oak_functions/proto/abi.proto

```diff
@@ -21,6 +21,20 @@ package oak.functions.abi;
 option java_multiple_files = true;
 option java_package = "oak.functions.abi";
 
+// ChannelStatus returned from a read, write, or wait on a channel.
+enum ChannelStatus {
+  CHANNEL_UNSPECIFIED = 0;
```

Comment:
The enum naming does not seem standard, see http://go/protodosdonts#do-include-an-unspecified-value-in-an-enum

Reply:
Done.